src/styles.scss

```
@@ -26,7 +26,7 @@ html {
 }
 
 html, body {
-  font-family: 'Roboto Mono';
+  font-family: 'Roboto';
```

We'd like to keep the font as is for now please.
```suggestion
  font-family: 'Roboto Mono';
```